src/extensions/CanvasWidget/CanvasWidget.tsx

```diff
@@ -101,13 +102,15 @@ const CanvasWidget = (props: Props) => {
         // or when the layers (prop) has changed (or if preventDefaultWheel has changed)
         // we set the canvas elements on the layers and schedule repaints
         if (!divElement) return
+        if (!layers) return
+        const stopScrollEvent = (e: Event) => {
+            e.preventDefault()
+        }
```

Comment:
Factored this out to a specific function in order to facilitate removing the event listener below. While that did not solve the problem, I left it as a separate item in case we want to try that some more.